code/weapon/beam.h

```diff
@@ -73,12 +72,16 @@ typedef struct beam_fire_info {
 	int point;									// for fighters, which point on the bank it is from
 	int bfi_flags;
 	char team;									// for floating beams, determines which team the beam is on
+	int burst_seed;								// used for sharing random targets if part of the same burst
+	float current_per_shot_rot;                         // for type 5 beams
+	float per_burst_rotation;                         // for type 5 beams
+	int burst_index;
```

Comment:
Does this need to be accounted for in multi? I assume that `burst_seed` doesn't (same as other PR), but `burst_index` and `per_burst_rotation` look like they might. It doesn't break anything in this case but I'd need to account for them in the packet fixes PR in order for type F beams to work properly in multi.

And `current_per_shot_rot` doesn't appear to be used anywhere.

Reply:
I think you might be right, on second thought I think I was misunderstanding how `beam_info` is actually used. And yes that bit is vestigal, thank you for pointing that out.

Reply:
`beam_fire_info` isn't properly accounted for in multi right now but I'm working to fix that. As things stand now the less general a beam is the less likely it is to produce the same result in multi as it does in single-player. We've tested that pretty much ever current beam functionality is working in multi now (with the fixes) so I just want to continue that trend.